composer.json

```diff
@@ -33,11 +33,13 @@
         "contao/imagine-svg": "^0.2.2 || ^1.0",
         "dantleech/phpcr-migrations-bundle": "^1.0",
         "doctrine/annotations": "^1.2",
-        "doctrine/common": "^2.7.1",
-        "doctrine/data-fixtures": "^1.1",
-        "doctrine/dbal": "^2.6",
+        "doctrine/collections": "^1.0",
+        "doctrine/cache": "^1.0",
+        "doctrine/data-fixtures": "^1.2.1",
+        "doctrine/dbal": "^2.6 || ^3.0",
         "doctrine/doctrine-bundle": "^1.10 || ^2.0",
         "doctrine/doctrine-fixtures-bundle": "^2.4 || ^3.0",
+        "doctrine/persistence": "^1.3 || ^2.0",
```

Comment:
Allow usage of `1.3` as it has class aliases from the old namespace to the new one. Projects can then as documented in the UPGRADE.md use the older version.